### PR TITLE
STORM-1493 Update Documentation for {{nimbus.seeds}}

### PR DESCRIPTION
**Pull request by HeartSaVioR**
Please note that I'm not fluent so you may want to correct syntax, nuance, and so on.

### CODE REVIEW

File: documentation/Setting-up-development-environment.md [hunk 1]
```diff
@@ -29,13 +29,5 @@ Installing a Storm release locally is only for interacting with remote clusters.
...
+nimbus.seeds: ["123.45.678.890"]
 ```
-
-Alternatively, if you use the [storm-deploy](https://github.com/nathanmarz/storm-deploy) project to provision Storm clusters on AWS, it will automatically set up your ~/.storm/storm.yaml file. You can manually attach to a Storm cluster (or switch between multiple clusters) using the "attach" command, like so:
-
-```
-lein run :deploy --attach --name mystormcluster
-```
-
-More information is on the storm-deploy [wiki](https://github.com/nathanmarz/storm-deploy/wiki)
```
Comment (redsanket) [inline documentation/Setting-up-development-environment.md]: why delete this, deprecated?

Comment (HeartSaVioR) [inline documentation/Setting-up-development-environment.md]: @redsanket 
Its last commit was Dec 11, 2013, and there're amount of issues saying "it doesn't work on ...". 
https://github.com/nathanmarz/storm-deploy/issues
So I think it is deprecated in point of Apache Storm's view.

**Comment by redsanket** [comment]
+1 LGTM

**Comment by ptgoetz** [comment]
+1

**Pull request merged**